.github/workflows/run_all_frameworks.yml

```diff
@@ -0,0 +1,68 @@
+name: Run All Frameworks
+
+on:
+  pull_request:
+    paths:
+      - 'amlb/**'
+      - 'frameworks/shared/**'
+      - '.github/workflows/run_all_frameworks.yml'
+      - '.github/runbenchmark/action.yml'
+      - 'runbenchmark.py'
+      - 'requirements.txt'
+
+
+jobs:
+  baseline:
+    name: Run Baseline on OpenML Iris
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            pip-
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install Requirements
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Run constantpredictor on openml iris
+        run: |
+          python runbenchmark.py constantpredictor openml/t/59 1h4c
```

Comment:
The reason I specified this is that;
 - it tests `openml` task specification which is not done in the validation tests (but it might be better as a separate test)
 - the `test` constraints don't test all folds, so I used `1h4c` , I should have probably done `-f 0 1 2 ... 9` instead to make that explicit. (I don't want the regular validation to be done on 10 folds, since that will incur an unreasonable workload).